packages/block-library/src/breadcrumbs/index.php

```diff
@@ -157,19 +157,53 @@ function block_core_breadcrumbs_maybe_add_paged( $breadcrumb_items ) {
 	return $breadcrumb_items;
 }
 
+/**
+ * Generates post type archive breadcrumb item.
+ *
+ * Returns the post type archive link item if the post type has archive enabled,
+ * otherwise returns null.
+ *
+ * @since 6.9.0
+ *
+ * @param string $post_type The post type name.
+ *
+ * @return string|null The archive breadcrumb item HTML, or null if not available.
+ */
+function block_core_breadcrumbs_get_post_type_archive_item( $post_type ) {
+	$post_type_object = get_post_type_object( $post_type );
+	if ( $post_type_object && $post_type_object->has_archive ) {
+		$archive_link = get_post_type_archive_link( $post_type );
+		if ( $archive_link ) {
+			return sprintf(
+				'<a href="%s">%s</a>',
+				esc_url( $archive_link ),
+				esc_html( $post_type_object->labels->name )
```

Comment:
There's also the `archives` label, which I agree isn't necessarily that broadly used. I wonder if we considered if we want to go with that, potentially.